### PR TITLE
fix: optimal ai create testcase chatcompletionmessage

### PR DESCRIPTION
**Pull request by wangzhuzhen**
#### What this PR does / why we need it:
Optimal ai create testcase chatcompletionmessages to get better result

#### Specified Reviewers:

/assign @sfwn

#### ChangeLog
Bugfix： Optimal ai create testcase chatcompletionmessages to get better result （优化 AI 生成测试用例的 Messges 输入以获取更好的生成效果）

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |     Optimal ai create testcase chatcompletionmessages to get better result         |
| 🇨🇳 中文    |   优化 AI 生成测试用例的 Messges 输入以获取更好的生成效果    |

#### Need cherry-pick to release versions?
/cherry-pick release/2.4

### CODE REVIEW

File: internal/pkg/ai-functions/handler/handler_ai_function_create_testcase.go [hunk 1]
```diff
@@ -147,7 +150,41 @@ func processSingleTestCase(ctx context.Context, factory functions.FunctionFactor
 			TestCaseID:      aiCreateTestCaseResponse.TestCaseID,
 		})
 	} else {
-		result, err := aiHandlerUtils.GetChatMessageFunctionCallArguments(ctx, factory, req, openaiURL, tp.Prompt, systemPrompt, callbackInput)
+
+		var (
+			f          = factory(ctx, "", req.GetBackground())
+			systemMsg0 = openai.ChatCompletionMessage{
+				Role:    openai.ChatMessageRoleSystem,
+				Content: "background: Your name is Erda AI Assistant. You are trained by Erda.",
```
Comment (sfwn) [inline internal/pkg/ai-functions/handler/handler_ai_function_create_testcase.go:158]: no need this message, it's auto-added by ai-proxy.

File: internal/pkg/ai-functions/handler/handler_ai_function_create_testcase.go [hunk 2]
```diff
@@ -22,6 +22,8 @@ import (
 	"strings"
 	"sync"
 
+	"github.com/sashabaranov/go-openai"
```
Comment (sfwn) [inline internal/pkg/ai-functions/handler/handler_ai_function_create_testcase.go:25]: regroup imports

**Review by sfwn — CHANGES_REQUESTED** [review]

File: internal/pkg/ai-functions/handler/handler_ai_function_create_testcase.go (hunk 1 above)
Comment (wangzhuzhen) [reply internal/pkg/ai-functions/handler/handler_ai_function_create_testcase.go]: OK

File: internal/pkg/ai-functions/handler/handler_ai_function_create_testcase.go (hunk 2 above)
Comment (wangzhuzhen) [reply internal/pkg/ai-functions/handler/handler_ai_function_create_testcase.go]: OK

**Review by sfwn — APPROVED** [review]

**Pull request merged**